CreatePackagesQuickStart.md

```diff
@@ -30,7 +30,7 @@ Here's a TL;DR quick start version of the package creating tutorial. Follow thes
 * **Test the package**
    * **Testing should probably be done on a Virtual Machine**
    * In your package directory, use:
-      * `choco install package-name -s '$pwd' -f` - powershell
+      * `choco install package-name -s "$pwd" -f` - powershell
```

Comment:
I mean that it doesn't expand it in the shell, causing spaces mess up argument parsing. It certainly passes the value to the exe. It's no big deal, this will work. If I can ever dig up official docs on the topic, I'll let you know :)